src/course-outline/configure-modal/BasicTab.jsx

```diff
@@ -21,23 +43,63 @@ const BasicTab = ({ releaseDate, setReleaseDate }) => {
           value={releaseDate}
           label={intl.formatMessage(messages.releaseDate)}
           controlName="state-date"
-          onChange={(date) => onChange(date)}
+          onChange={(date) => onChangeReleaseDate(date)}
         />
         <DatepickerControl
           type={DATEPICKER_TYPES.time}
           value={releaseDate}
           label={intl.formatMessage(messages.releaseTimeUTC)}
           controlName="start-time"
-          onChange={(date) => onChange(date)}
+          onChange={(date) => onChangeReleaseDate(date)}
         />
       </Stack>
+      {
+        isSubsection ? (
+          <div>
+            <h3 className="mt-3"><FormattedMessage {...messages.grading} /></h3>
```

Comment:
Please make all headings inside the configure modal to `h5` and add class `text-gray-700`.

Also, change `mt-3` to `mt-4` here to increase top margin.